src/plugins/optimize-graph-plugin.js

```diff
@@ -373,14 +390,15 @@ function hoistTransitiveImports(graph) {
 					appendCode += '\n' + DEFAULT_STYLE_LOAD_IMPL;
 				}
 				if (DEBUG) console.log(`Preloading CSS for import(${spec}): ${css}`);
-				preloads.push(...css.map(f => `${meta.styleLoadFn}(${JSON.stringify(posix.join(graph.publicPath, f))})`));
+				preloads.push(...css.map(f => `${meta.styleLoadFn}(${toImport(graph.publicPath, f, true)})`));
 			}
 
 			const js = deps.js.get(spec);
 			if (js && js.length) {
 				if (DEBUG) console.log(`Preloading JS for import(${spec}): ${js}`);
 				preloads.push(
 					...js.map(f => {
+						// TODO: should have `toImport(...)` somewhere here
```

Comment:
TBH I'm not sure what's really going on with the next line. But in any event, its `graph.publicPath` isn't yet accounting for the fact that `publicPath` may be an external address.

I couldn't figure out how to reach this block of code, so couldn't confidently update this.

Reply:
ah - these are Rollup-generated `import()` statements and already have the public path prepended.

Reply:
Ok, so they're safe to ignore?

Reply:
I'm still concerned about this. Even if they're already `publicPath`d by Rollup, we're still doing this, which is causing all the issues with external/protocol'd `publicPath` values:

```js
posix.join(graph.publicPath, f)
```

Reply:
Ah, yeah good point. One option might be to just skip the conversion to relative if publicPath contains `//` ?